templates/cluster-template.yaml

```diff
@@ -65,7 +65,7 @@ spec:
         content: |
           # Restrict key exchange, cipher, and MAC algorithms, as per sshaudit.com
           # hardening guide.
-          KexAlgorithms curve25519-sha256,curve25519-sha256@libssh.org,diffie-hellman-group16-sha512,diffie-hellman-group18-sha512,diffie-hellman-group-exchange-sha256
+          KexAlgorithms sntrup761x25519-sha512@openssh.com,curve25519-sha256,curve25519-sha256@libssh.org,gss-curve25519-sha256-,gss-group16-sha512-,diffie-hellman-group16-sha512,gss-group16-sha512-,diffie-hellman-group18-sha512,diffie-hellman-group-exchange-sha256
```

Comment:
Hm, `gss-group16-sha512-` is duplicated and therefore differs from https://sshaudit.com/hardening_guides.html#ubuntu_22_04_lts. How can that happen, didn't you just copy it?

Reply:
Now `gss-group16-sha512-` is missing completely. So is `gss-curve25519-sha256-`.

Reply:
Still missing

Reply:
Now all `KexAlgorithms` values are there, but there are shuffled :D 

Apart from that, it lgtm